lib/tasks/first_timer_tag.rake

```diff
@@ -0,0 +1,30 @@
+namespace :first_timer_tag do
+  desc 'Script that goes back and checks if the first post of a user has
+  first-timer poster if not, it adds it'
+
+  task add_poster_tag:  :environment do
+    p 'Finding first timer tag'
+    find_first_time_tag
+    p 'Updating first timer tag to all users first posts...'
+    find_and_update
+    p 'Done...'
+  end
+end
+
+def find_and_update
+  arr = []
+  User.all.each do |user|
```

Comment:
this could be a big query ... can we join `nodes` where `node.status = 1`, as we do in other parts of the code? That'll make the query faster -- you can probably find an example in `/db/migrate/...`

Reply:
Hmm... can't find a clear example, but it might be like:

```ruby
User.joins(:node).where('node.status = ?', 1).each do |user|
```